doc/release/1.0.0-notes.rst

```diff
@@ -136,6 +136,36 @@ an array with a single element.
 Backwards incompatible changes
 ==============================
 
+The following deprecated functions have been removed from `scipy.stats`:
+``betai``, ``chisqprob``, ``f_value``, ``histogram``, ``histogram2``,
+``pdf_fromgamma``, ``signaltonoise``, ``square_of_sums``, ``ss`` and
+``threshold``.
+
+The following deprecated functions have been removed from `scipy.stats.mstats`:
+``betai``, ``f_value_wilks_lambda``, ``signaltonoise`` and ``threshold``.
+
+The deprecated ``a`` and ``reta`` keywords have been removed from
+`scipy.stats.shapiro`.
+
+The deprecated functions ``sparse.csgraph.cs_graph_components`` and
+``sparse.linalg.symeig``.  have been removed from `scipy.sparse`:
```

Comment:
weird period after `symeig` and colon at the end

Reply:
thanks, fixed